pkg/kubelet/server/stats/summary_sys_containers_windows.go

```diff
@@ -28,7 +28,9 @@ import (
 )
 
 func (sp *summaryProviderImpl) GetSystemContainersStats(nodeConfig cm.NodeConfig, podStats []statsapi.PodStats, updateStats bool) (stats []statsapi.ContainerStats) {
-	stats = append(stats, sp.getSystemPodsCPUAndMemoryStats(nodeConfig, podStats, updateStats))
+	containerStats := sp.getSystemPodsCPUAndMemoryStats(nodeConfig, podStats, updateStats)
+	sp.addSystemPodsSwapStats(podStats, &containerStats)
```

Comment:
any reason to not keep it under the feature gate check?

Reply:
also based on https://github.com/kubernetes/kubernetes/pull/118865/files#r1260089098 we may need to make it OS-specific